tests/test_open_images_format.py

```diff
@@ -110,6 +111,42 @@ def test_can_save_and_load_image_with_arbitrary_extension(self):
 
             compare_datasets(self, dataset, parsed_dataset, require_images=True)
 
+    @mark_requirement(Requirements.DATUM_274)
+    def test_inplace_save_writes_only_updated_data(self):
+        dataset = Dataset.from_iterable([
+            DatasetItem('a', subset='s', image=np.ones((2, 1, 3)),
```

Comment:
Consider having an unchanged subset and unchanged items in modified subsets.

Reply:
Alright, implemented (I also gave the subsets better names).